lib/ansible/modules/source_control/github_release.py

```diff
@@ -120,6 +178,20 @@ def main():
         else:
             module.exit_json(tag=None)
 
+    if action == 'create_release':
+        if tag is None:
+            module.fail_json(msg="Specify tag in order to create a release.")
+        release_exists = repository.release_from_tag(tag)
+        if release_exists:
+            module.exit_json(
+                skipped=True, msg="Release for tag %s already exists." % tag)
+
+        release = repository.create_release(
+            tag, target, name, body, draft, prerelease)
+        if release:
+            module.exit_json(tag=release.tag_name)
+        else:
+            module.exit_json(tag=None)
 
 from ansible.module_utils.basic import *
```

Comment:
Since you are in the and testing this anyway, could you please update this from a wildcard `*` import to named imports

https://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#python-imports